README.md

```diff
@@ -1,17 +1,26 @@
 This repository contains resources for configuring kubernetes with calico networking plugin using Solar.
 
-Setup:
+Vagrant setup:
 
-1. You need solar dev env for now.
-2. Put Vagrantfile_solar instead of default Vagrantfile that is included in solar.
-3. Add fc23 vagrant box: `vagrant box add fc23 Fedora-Cloud-Base-Vagrant-23-20151030.x86_64.vagrant-libvirt.box --provider libvirt  --force`
-4. Change boxes in `vagrant-settings.yaml` like:
-
-    master_image: solar-master
-    master_image_version: null
-    slaves_image: fc23
-    slaves_image_version: null
-
-5. Login on solar-dev, link this directory as /var/lib/solar/resources/k8s (or solar repository import -l ...)
-6. Install python on solar-dev* (sudo dnf install python). It's required for ansible
-7. `setup_k8s.py` is a naive script that adds resources. Then you can proceed with normal solar steps.
+1. Clone [solar](https://github.com/openstack/solar)
+2. Copy Vagrantfile_solar from this repo to solar Vagrantfile
+3. Add fc23 vagrant box:
+	* libvirt: `vagrant box add fc23 Fedora-Cloud-Base-Vagrant-23-20151030.x86_64.vagrant-libvirt.box --provider libvirt  --force`
+	* virtualbox: `vagrant box add fc23 Fedora-Cloud-Base-Vagrant-23-20151030.x86_64.vagrant-virtualbox.box --provider virtualbox --force`
+4. ensure that vagrant-settings.yaml contains these values:(slave count na 2)
+    * slaves_count: 2
+    * master_image: solar-master
+    * master_image_version: null
+    * slaves_image: fc23
+    * slaves_image_version: null
+5. vagrant up
+6. Copy, link or clone this repo to solar-dev VM into k8s folder
+8. solar repo import -l k8s
+9. ./setup-k8s.py
+10. solar changes stage
+11. solar changes process
+12. solar orch run-once
+13. watch solar orch report
+14. (if something fails it's possible that k8s didn't start yet, let's try again) solar orch retry last
```

Comment:
Can we remove this comment for now ? we should ensure that there is no such failure. I added some wait_port stuff yesterday. Do you remember exactly what else fails?